webpack/loaders.js

```diff
@@ -0,0 +1,22 @@
+exports.typingsForCssModulesLoader = (prod = false) => ({
```

Comment:
this takes the place of the `css-loader` when loading styles from `/src/component` since we apply types to our styles.